gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Mon Feb 11 23:37:42 CET 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-rc-4-all.zip
```

Comment:
there is the release version now for 6.7 -> `gradle-6.7-all.zip` 😉

Reply:
I know! I hoped to get away with it! :)